types/src/lib.rs

```diff
@@ -38,6 +38,7 @@ mod slugs;
 mod tag;
 mod translation;
 pub mod user;
+pub mod comment;
```

Comment:
I've noticed my imports from this module have to be qualified an extra time when they are imported (`dailp::comment::whatever`) is that because I used `pub mod ...`? Should I get rid of `pub`?

Reply:
Yeah, we keep everything package private. It also helps prevent crossing wires in other rust code which is nice